pkg/cmd/label/delete_test.go

```diff
@@ -102,11 +102,10 @@ func TestDeleteRun(t *testing.T) {
 					httpmock.StatusStringResponse(204, "{}"),
 				)
 			},
-			askStubs: func(q *prompt.AskStubber) {
+			askStubs: func(as *prompt.AskStubber) {
 				// TODO: survey stubber doesn't have WithValidator support
 				// so this always passes regardless of prompt input
-				//nolint:staticcheck // SA1019: q.StubOne is deprecated: use StubPrompt
-				q.StubOne("OWNER/REPO")
+				as.StubPrompt("Type test to confirm deletion:").AnswerWith("test")
```

Comment:
@samcoe does this make the TODO comment just above that I copied from another command moot? Should it be removed if so?

Reply:
The TODO is still valid even with using the new functions.